SwrveSDK/src/main/java/com/swrve/sdk/locationcampaigns/LocationCampaignFilter.java

```diff
@@ -35,7 +34,8 @@
         try {
             wakeLock.acquire();
 
-            ((SwrveBase)(SwrveSDKBase.getInstance())).initLocationCampaigns(); // TODO: presumption for FA is that location campaigns have been refreshed from ABTS by starting the app. Fix for GA.
+            // TODO: presumption for FA is that location campaigns have been refreshed from ABTS by starting the app. Also that certain variables have been initialised. For FA hack this. For GA fix this.
```

Comment:
No, onCreate will not be called from application class.

We will be refactoring the init method of the sdk so that a lot of this code should be initialised in the createInstance. Not doing this for FA though.